drivers/flash/Kconfig

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config FLASH_RPC
+	bool "Flash over RPC [EXPERIMENTAL]"
+	select NRF_RPC
+	select NRF_RPC_CBOR
+	help
```

Comment:
Shouldn't this depend on `FLASH`?

Reply:
It should depend on flash only if it's compiled for the network core I'll try to fix this.